forge-gui/res/cardsfolder/upcoming/bennie_bracks_zoologist.txt

```diff
@@ -0,0 +1,9 @@
+Name:Bennie Bracks, Zoologist
+ManaCost:3 W
+Types:Legendary Creature Elf Druid
+PT:3/2
+K:Convoke
+T:Mode$ Phase | Phase$ End of Turn | CheckSVar$ X | SVarCompare$ GE1 | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ At the beginning of each end step, if you created a token this turn, draw a card.
+SVar:TrigDraw:DB$ Draw | Defined$ You | NumCards$ 1

```

Comment:
`Defined$ You` is default for any Defined
`NumCards$ 1` is default for Draw effect

Those params are not doing any harm, and you don't *need* to remove them, but you could if you wanted to.

Reply:
Little nervous about this one but I trust you! :D

Reply:
You don't have to take my word for it; I wouldn't fault you for testing it 😁